### PR TITLE
add window setAppDetails feature to modify windows user task in taskbar

### LINKED ISSUE #154: A question of the taskbar under the Win10
I was building my App using `go-astilectron-bootstrap` and `go-astilectron-bundler`, but something wrong with win10 about the taskbar, when I run the App and right-click on the taskbar, then I found that the `Electron` app in the menu. Are some options set incorrectly?

### PR DESCRIPTION
**Pull request by fregie**
Fixes #154 
add window setAppDetails feature to modify windows user task
like this:
image
BTW,don't forget to merge this in astilectron:
https://github.com/asticode/astilectron/pull/15

### CODE REVIEW

File: window.go [hunk 1]
```diff
@@ -116,9 +117,19 @@ type WindowOptions struct {
 	Y                      *int            `json:"y,omitempty"`
 
 	// Additional options
-	Custom *WindowCustomOptions `json:"custom,omitempty"`
-	Load   *WindowLoadOptions   `json:"load,omitempty"`
-	Proxy  *WindowProxyOptions  `json:"proxy,omitempty"`
+	Custom     *WindowCustomOptions `json:"custom,omitempty"`
+	Load       *WindowLoadOptions   `json:"load,omitempty"`
+	Proxy      *WindowProxyOptions  `json:"proxy,omitempty"`
+	AppDetails *WindowAppDetails    `json:"appDetails,omitempty"`
```
Comment (asticode) [inline window.go]: Could you move `AppDetails` above `Custom` so that additional options are sorted alphabetically?

File: window.go [hunk 2]
```diff
@@ -116,9 +117,19 @@ type WindowOptions struct {
...
+}
+
+// WindowAppDetails represents window app details
+type WindowAppDetails struct {
```
Comment (asticode) [inline window.go]: Could you add this link `https://github.com/electron/electron/blob/v4.0.1/docs/api/browser-window.md#winsetappdetailsoptions-windows` as an additional commentary below `// WindowAppDetails represents window app details`?

File: window.go [hunk 3]
```diff
@@ -197,6 +208,9 @@ func newWindow(o Options, p Paths, url string, wo *WindowOptions, c *asticontext
 	if wo.Title == nil && o.AppName != "" {
 		wo.Title = PtrStr(o.AppName)
 	}
+	if runtime.GOOS != "windows" {
```
Comment (asticode) [inline window.go]: Can you remove this? I'd rather have this logic in the JS

**Comment by fregie** [comment]
Already modified as requested

**Comment by asticode** [comment]
I've created a `v0.31.0` tag for `astilectron`, can you update the `astilectron` version?

**Comment by fregie** [comment]
done.

**Pull request merged**